openproblems/tasks/_batch_integration/batch_integration_embed/__init__.py

```diff
@@ -5,8 +5,7 @@
 from . import metrics
 
 _task_name = "Batch integration embed"
-_task_summary = """ Removing batch effects while preserving biological
-                    variation in single-cell data embeddings """
+_task_summary = """ Removing batch effects while preserving biological variation """
```

Comment:
We can't say "in embeddings" here? Otherwise this task blurb is for the full data.

Reply:
maximum length of 80(?) characters doesn't allow for the long version

Reply:
`Removing batch effects while preserving biological variation (embedding output)` works

Reply:
true, thanks for the suggestion